### DISCUSSION TITLE
Azanyum Sazanyum sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#213] [Soru Cevap]
Evrenin gizemli bir yerinde Diven diye bir gezegende Azanyum ve Sazanyum adlı iki türlü yaratık yaşıyordu. Her yılın son günü her Azanyum, bir sazanyum ve bir azanyum yaratacak şekilde bölünüyordu. Bir sazanyum ise bir azanyum iki sazanyuma bölünmekteydi. Bu gezegende hayatın tek bir azanyum ile başladığı bilinmekteydi.
İncemeler şunu gösterdiki Sazanyum/Azanyum oranı 5/3 ü aştığında bu gezegende yaşam son bulacak. Sizce Diven gezegeninde yaşam bir gün son bulur mu?

**Comment by OmerB2** [comment]
```python
a=1
s=0
for y in range(1,101):
    az=a+s
    sa=a+s*2
    a=a+az
    s=s+sa
    if s/a>=5/3:
        print("Diven gezegeninde yaşam",y,"yılında son bulur.")
        break
    print(y,"Yılında",a,"azanyum",s,"sazanyum olur.")
if s/a!=5/3:
    print("Diven gezegeninde yaşam son bulmaz.")
#bu kod sadece yüz kere deniyor ama isterseniz arttırabilirsiniz.

```

**Reply by sonsuzus** [reply]
5/3 aşma durumlarını eşit değil de `>=` ile yapsan daha doğru olurdu. Çünkü 5/3 den büyük olduğu halde eşit olmayabilir.

**Reply by OmerB2** [reply]
tamamdır hocam
